edgelet/iotedge/src/check/checks/check_sockets.rs

```diff
@@ -0,0 +1,134 @@
+use std::os::unix::prelude::{MetadataExt, PermissionsExt};
+
+use edgelet_core::UrlExt;
+use edgelet_settings::RuntimeSettings;
+use failure::{Context, ResultExt};
+use nix::unistd::{Gid, Group};
+use url::Url;
+
+use crate::check::{Check, CheckResult, Checker, CheckerMeta};
+
+const MANAGEMENT_SOCKET_DEFAULT_PERMISSION: u32 = 0o660;
```

Comment:
how did to confirm these permissions? is it logged somewhere?

Reply:
Yes, They are const in workload.rs and management.rs. I prefered redefining them here instead of taking dependencies on different libraries only for those 2 constants.